src/fileReader.ts

```diff
@@ -13,23 +13,43 @@ export async function* gameChunks(
   });
 
   let metadata: string[] = [];
+  let moves: string = '';
+  let ignoreGame: boolean = false;
   for await (const line of reader) {
-    // metadata & move lines chunked as a single game
-    if (line.startsWith('[')) {
-      metadata.push(line);
-    } else if (!line) {
-      // empty line, do nothing
+    if (line.startsWith('[Event')) {
+      // Start of a new game
+      ignoreGame = false;
+      metadata = [line];
+    } else if (line.startsWith('[')) {
+      if (!ignoreGame) {
+        metadata.push(line);
+      }
     } else if (line.startsWith('1.')) {
```

Comment:
the ignoreGame variable was added because when decompressing the original file into different files, inevitably games were getting cut off. So I wanted to ignore those games rather than trying to piece them together (the data loss is minimal relatively speaking; ~1500 files were created from the 10mil games analysis, which means around 1500 games got cut off)

Reply:
will accept your suggestion but idk if necessary

Reply:
we could probably solve this by decompressing with extra bits (if this is supported) and then slicing off the end at a known valid point. Or we just ignore the last 3 lines of every file